rotkehlchen/tests/db/test_locations.py

```diff
@@ -19,6 +21,7 @@
 )
 
 
+@pytest.mark.parametrize('use_clean_caching_directory', [True])
```

Comment:
Okay so looking at this being put everywhere I am starting to suspect there is some sort of fixture we have that works fine and implcitly clears something in Unix, but does not work in Windows. Can't explain the need to add this everywhere otherwise.